ui/secret-admin/package.json

```diff
@@ -0,0 +1,113 @@
+  "source": "src/index.js",
+  "module": "build/index.js",
+  "license": "Apache-2.0",
+  "private": true,
+  "devDependencies": {
+    "@babel/core": "^7.20.2",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.24.1",
+    "@svgr/core": "^7.0.0",
+    "@svgr/plugin-jsx": "^7.0.0",
+    "@tanstack/react-query": "4.28.0",
+    "@testing-library/dom": "^8.19.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^14.4.3",
+    "assert": "^2.0.0",
+    "autoprefixer": "^10.4.2",
+    "babel-jest": "^29.3.1",
+    "babel-plugin-transform-import-meta": "^2.2.0",
+    "communicator": "https://assets.juno.global.cloud.sap/libs/communicator@2.2.6/package.tgz",
+    "esbuild": "^0.19.12",
+    "jest": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
+    "juno-ui-components": "https://assets.juno.global.cloud.sap/libs/juno-ui-components@2.13.5/package.tgz",
+    "luxon": "^2.3.0",
+    "postcss": "^8.4.21",
+    "postcss-url": "^10.1.3",
+    "prop-types": "^15.8.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-test-renderer": "18.2.0",
+    "sapcc-k8sclient": "^1.0.2",
+    "sass": "^1.60.0",
+    "shadow-dom-testing-library": "^1.7.1",
+    "tailwindcss": "^3.3.1",
+    "url-state-provider": "https://assets.juno.global.cloud.sap/libs/url-state-provider@1.3.2/package.tgz",
+    "util": "^0.12.4",
+    "zustand": "4.3.7"
+  },
+  "peerDependencies": {
+    "juno-ui-components": "*",
+    "luxon": "^2.3.0",
+    "prop-types": "^15.8.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "url-state-provider": "*",
+    "utils": "https://assets.juno.global.cloud.sap/libs/utils@1.1.6/package.tgz",
+    "zustand": "4.3.7"
+  },
+  "scripts": {
+    "test": "jest",
+    "start": "NODE_ENV=development node esbuild.config.js --serve --watch",
+    "build": "NODE_ENV=production node esbuild.config.js"
+  },
+  "importmapExtras": {
+    "zustand/middleware": "4.3.7"
+  },
+  "appProps": {
+    "authIssuerUrl": {
+      "value": "https://endpoint_url_of_the_openid_provider.com",
+      "type": "required",
+      "description": "Endpoint URL of the OpenID provider"
+    },
+    "authClientId": {
+      "value": "tbd",
+      "type": "required",
+      "description": "OIDC client id. "
+    },
+    "mockAuth": {
+      "value": false,
+      "type": "optional",
+      "description": "mock the OIDC data, allowed values are 'true', 'false' (default), or json (pure or base64 encoded)"
+    },
+    "theme": {
+      "value": "theme-dark",
+      "description": "Override the default theme. Possible values are theme-light or theme-dark (default)"
+    },
+    "embedded": {
+      "value": "false",
+      "description": "Set to true if app is to be embedded in another existing app or page, like e.g. Elektra.  If set to true the app won't render a page header/footer and instead render only the content"
```

Comment:
We probably need to remove the example from Elektra. It is not related in this context.